internal/sshd/client.go

```diff
@@ -242,7 +242,10 @@ func (c *Client) handleServerInfo(proxy conf.Proxy, req *ssh.Request) {
 	var hostURL string
 	switch c.protocol {
 	case "tcp":
-		host := proxy.Domain
+		host := proxy.TCP.Domain
+		if host == "" {
+			host = proxy.Domain
+		}
```

Comment:
We have this nice helper available 😁 

https://github.com/pgrok/pgrok/blob/eed3ebf3d483231f4177c8b6f20f8beb08f10990/internal/strutil/strutil.go#L44-L45

Reply:
thanks!